### DISCUSSION TITLE
Remote Filesystem viewer

### DISCUSSION THREAD

**Discussion by h-bag** [GitSquared/edex-ui#1109] [Ideas]
I'm not too sure how easy this would be to implement (if it's fairly simple I could try doing it myself - I could use the practice), but it'd be pretty cool if the file browser was able to show the contents of the current directory on the remote host you're connected to. Same as how it does for your local system, except for whatever you're connected to via SSH.

**Comment by GitSquared** [comment]
I actually worked on this idea for a while, but ended up dropping it to focus on more "important" projects.
There's a bunch of different ways to do this, and none of them are really easy. The approach i hacked on was writing an edex-server program which the client app would detect, connect to (securely... which is a whole other topic) and proxy stuff like system information queries/filesystem directory list through.
